README.md

```diff
@@ -34,9 +34,10 @@ With `gatsby-transformer-cloudinary` you may:
 
 Use assets hosted by Cloudinary together with Gatsby's Image component:
 
-- The `gatsbyImageData` resolver is added to each GraphQLType configured.
+- You add the `gatsbyImageData` resolver to each GraphQLType configured.
```

Comment:
You don't, the plugin does. 

So maybe "the plugin adds"

Reply:
👍